project-management/src/main/java/life/qbic/projectmanagement/application/rawdata/RawDataService.java

```diff
@@ -0,0 +1,174 @@
+              proteomicsMeasurement -> proteomicsMeasurement.measurementId()
+                  .equals(rawData.measurementId()))
+          .findAny().orElseThrow();
+      List<RawDataSampleInformation> sampleInformation = sampleInformationService.retrieveSamplesByIds(
+              measurementWithData.measuredSamples()).stream()
+          .map(sample -> new RawDataSampleInformation(sample.sampleCode(), sample.label()))
+          .toList();
+      return new RawData(measurementWithData.measurementCode(), sampleInformation,
+          rawData.registrationDate());
+    }).toList();
+  }
+
+  @PostAuthorize(
+      "hasPermission(#projectId, 'life.qbic.projectmanagement.domain.model.project.Project', 'READ') ")
+  public Collection<RawData> findNGSRawData(String filter,
+      ExperimentId experimentId,
+      int offset, int limit,
+      List<SortOrder> sortOrder, ProjectId projectId) {
+    var measurements = retrieveNGSMeasurementsForExperiment(experimentId, filter, offset, limit,
+        sortOrder);
+    var measurementIds = measurements.stream().map(NGSMeasurement::measurementId).toList();
+    var rawDomainData = rawDataLookupService.queryRawDataByMeasurementIds(filter, measurementIds,
+        offset, limit, sortOrder);
+    return rawDomainData.stream().map(rawData -> {
+      NGSMeasurement measurementWithData = measurements.stream()
+          .filter(ngsMeasurement -> ngsMeasurement.measurementId().equals(rawData.measurementId()))
+          .findAny().orElseThrow();
+      List<RawDataSampleInformation> sampleInformation = sampleInformationService.retrieveSamplesByIds(
+              measurementWithData.measuredSamples()).stream()
+          .map(sample -> new RawDataSampleInformation(sample.sampleCode(), sample.label()))
+          .toList();
+      return new RawData(measurementWithData.measurementCode(), sampleInformation,
+          rawData.registrationDate());
+    }).toList();
+  }
+
+
+  public RawDataFileInformation findRawDataFileInformationForMeasurementCode(
+      MeasurementCode measurementCode) {
+    //Todo implement OpenBis connection?
+    return new RawDataFileInformation("Q0000_primary_results.zip", "78GB", "20",
+        "120EA8A25E5D487BF68B5F7096440019");
+  }
+
+  private List<NGSMeasurement> retrieveNGSMeasurementsForExperiment(ExperimentId experimentId,
+      String filter, int offset, int limit,
+      List<SortOrder> sortOrder) {
+    var result = sampleInformationService.retrieveSamplesForExperiment(experimentId);
+    var samplesInExperiment = result.getValue().stream().map(Sample::sampleId).toList();
+    /*We need to cast the id property manually*/
+    var measurementSortOrder = sortOrder.stream().map(sortOrder1 -> new SortOrder("id", true))
+        .toList();
+    return measurementLookupService.queryNGSMeasurementsBySampleIds(filter,
+        samplesInExperiment, offset, limit, measurementSortOrder);
+  }
+
+  private List<ProteomicsMeasurement> retrieveProteomicsMeasurementsForExperiment(
```

Comment:
Since we want to show the measurements and raw data for each domain as a seperate tab, we need to see if there are measurements registered for each domain to load the tabs in the tabsheet individually